server/util.js

```diff
@@ -27,6 +28,12 @@ const createAuthResponseObj = async (user) => {
     };
 };
 
+const createChefProfile = async (id) => {
```

Comment:
Can we name this find or fetch rather than create?

Reply:
Yes, that makes more sense.